src/app/myworkflows/my-workflow/my-workflow.component.ts

```diff
@@ -111,16 +111,30 @@ export class MyWorkflowComponent extends MyEntry implements OnInit {
       if (user) {
         this.user = user;
         this.stateService.setRefreshMessage('Fetching workflows');
-        combineLatest(this.usersService.userWorkflows(user.id).pipe(first()), this.workflowsService.sharedWorkflows().pipe(first()))
-          .subscribe(([workflows, sharedWorkflows]) => {
-            if (workflows && sharedWorkflows) {
-              this.workflowService.setWorkflows(workflows);
-              this.workflowService.setSharedWorkflows(sharedWorkflows);
-              this.stateService.setRefreshMessage(null);
-            }
-          }, (error: any) => {
+        let count = 2;
```

Comment:
A bit confusing

I'd prefer two booleans (e.g. `userWorkflowsFound` and `sharedWorkflowsFound`) and test with `userWorkflowsFound && sharedWorkflowsFound` instead of `count === 0` even though its lengthier

Reply:
could change it to 2 booleans, but "found" may be misleading

Reply:
@garyluu fair enough, use whatever variable name is accurate yet descriptive